### PR TITLE
NATIVE-431: Search feedback fails if empty query

### PR DESCRIPTION
**Pull request by steffenkleinle**

### CODE REVIEW

File: src/endpoints/createFeedbackEndpoint.js [hunk 1]
```diff
@@ -46,16 +46,16 @@ export default (baseUrl: string): Endpoint<ParamsType, {}> => new EndpointBuilde
   .withParamsToBodyMapper((params: ParamsType): FormData => {
     const formData = new FormData()
     formData.append('rating', params.isPositiveRating ? POSITIVE_RATING : NEGATIVE_RATING)
-    if (params.permalink) {
+    if (params.permalink !== undefined) {
```
Comment (maxammann) [inline src/endpoints/createFeedbackEndpoint.js:49]: I think this check should be `params.permalink !== undefined && params.permalink !== null`. What happens if null is sent to the cms?

@timoludwig

**Review by maxammann — CHANGES_REQUESTED** [review]

File: src/endpoints/createFeedbackEndpoint.js (hunk 1 above)
Comment (timobrembeck) [reply src/endpoints/createFeedbackEndpoint.js:49]: It makes no difference for the cms whether the permalink parameter is `null` or just not sent at all.

Comment (klinzo) [reply src/endpoints/createFeedbackEndpoint.js]: Also, `null` is not allowed by flow

Comment (steffenkleinle) [reply src/endpoints/createFeedbackEndpoint.js:49]: Also, null is not an allowed type by flow.

**Review by michael-markl — APPROVED** [review]

**Review by maxammann — APPROVED** [review]

**Pull request merged**